internal/server/htmlui_embed.go

```diff
@@ -1,13 +1,12 @@
-// +build embedhtml
```

Comment:
Let's bring back a way of building without the embedded HTML. We can default to including the HTML UI, and have a build flag to build without the embedded HTML.